### PR TITLE
Add warning comment

### PR DESCRIPTION
**Pull request by Doodoune**
The demo env is a duplication of demo ref. 
It's updated every hour at 42 minutes.

Beware to not update demo ref during this duplication !

### CODE REVIEW

File: src/Release/Command/TagCommand.php [hunk 1]
```diff
@@ -158,7 +158,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $matches['major']
             ),
             '',
-            'Then you need to update the public demo (only if you tagged the last major version):',
+            'Then you need to update the public demo ref (only if you tagged the last major version):',
+            '/!\ Warning ! The demo env is a duplication of demo ref. It\'s updated every hour at 42 minutes.',
```
Comment (damien-carcel) [inline src/Release/Command/TagCommand.php]: is a duplicate

File: src/Release/Command/TagCommand.php [hunk 2]
```diff
@@ -158,7 +158,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+            'Beware to not update demo ref during this duplication !',
```
Comment (damien-carcel) [inline src/Release/Command/TagCommand.php]: Beware not to

Comment (damien-carcel) [reply src/Release/Command/TagCommand.php]: during the duplication process

File: src/Release/Command/TagCommand.php (hunk 1 above)
Comment (Doodoune) [reply src/Release/Command/TagCommand.php]: done thanks ! :+1:

File: src/Release/Command/TagCommand.php (hunk 2 above)
Comment (Doodoune) [reply src/Release/Command/TagCommand.php]: done thanks ! :+1:

**Pull request merged**